### ISSUE TITLE
Need to define indicator image of a node in run-time

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bctpl/tree-view-list-android#20]
```
Thank you for exelent treeview.

In my tree-view nodes have 4 different states - 2 collapsed and 2 expanded 
depending on external reasons. Currently I can use only 2 drawable for 
indicator states, but if AbstractTreeViewAdapter.getDrawable would be protected 
instead of private, I'd have an ability to set needed image in appopriate 
moment (when node have been clicked) in my TreeAdapter.

```

Original issue reported on code.google.com by `DmLa...@gmail.com` on 17 Jan 2012 at 7:19

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e9de4bed74d2.
```

Original comment by `jarek.po...@polidea.pl` on 23 Jan 2012 at 5:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thank you!
```

Original comment by `DmLa...@gmail.com` on 23 Jan 2012 at 5:31
